moveit_ros/benchmarks/examples/demo_panda_predefined_poses.yaml

```diff
@@ -16,7 +16,7 @@ benchmark_config:
         group: panda_arm      # Required
         timeout: 10.0
         output_directory: /tmp/moveit_benchmarks/
-        predefined_poses_group: panda_arm_hand # Group where the predefined poses are specified
+        predefined_poses_group: panda_arm # Group where the predefined poses are specified
```

Comment:
odd, the `predefined_poses_group` was set to this on purpose, but I didn't find any related changes skimming the config packages.